doc/cli.md

```diff
@@ -0,0 +1,230 @@
+```
+% ./bin/persephone.sh validate \
+  --profile sample_profiles/calibration.jsonld \
+  --statement sample_statements/calibration_1.json \
+  --all-valid
+```
+This will run and print the following message to stdout before exiting with code 1:
+```
+----- Statement Validation Failure -----
+Template ID:  https://xapinet.org/xapi/yet/calibration/v1/templates#activity-2
+Statement ID: 00000000-4000-8000-0000-000000000000
+
+Template Verb property was not matched.
+ template Verb:
+   https://xapinet.org/xapi/yet/calibration/v1/concepts#didnt
+ statement Verb:
+   https://xapinet.org/xapi/yet/calibration/v1/concepts#did
+
+Template rule was not followed:
+  {:any [\"https://xapinet.org/xapi/yet/calibration/v1/concepts#activity-2\"],
+   :location \"$.object.id\",
+   :presence \"included\"}
+ failed 'any' property: evaluated values must include some values given by 'any'
+ statement values:
+   https://xapinet.org/xapi/yet/calibration/v1/concepts#activity-1
+
+Template rule was not followed:
+  {:any [\"Activity 2\"],
+   :location \"$.object.definition.name.en-US\",
+   :presence \"included\"}
+ failed 'any' property: evaluated values must include some values given by 'any'
+ statement values:
+   Activity 1
+
+Template rule was not followed:
+  {:any [\"The second Activity\"],
+   :location \"$.object.definition.description.en-US\",
+   :presence \"included\"}
+ failed 'any' property: evaluated values must include some values given by 'any'
+ statement values:
+   The first Activity
+
+----- Statement Validation Failure -----
+Template ID:  https://xapinet.org/xapi/yet/calibration/v1/templates#activity-3
+Statement ID: 00000000-4000-8000-0000-000000000000
+
+Template rule was not followed:
+  {:any [\"https://xapinet.org/xapi/yet/calibration/v1/concepts#activity-3\"],
+   :location \"$.object.id\",
+   :presence \"included\"}
+ failed 'any' property: evaluated values must include some values given by 'any'
+ statement values:
+   https://xapinet.org/xapi/yet/calibration/v1/concepts#activity-1
+
+Template rule was not followed:
+  {:any [\"Activity 3\"],
+   :location \"$.object.definition.name.en-US\",
+   :presence \"included\"}
+ failed 'any' property: evaluated values must include some values given by 'any'
+ statement values:
+   Activity 1
+
+Template rule was not followed:
+  {:any [\"The third Activity\"],
+   :location \"$.object.definition.description.en-US\",
+   :presence \"included\"}
+ failed 'any' property: evaluated values must include some values given by 'any'
+ statement values:
+   The first Activity
+
+-----------------------------
+Total errors found: 7
+```
+
+If we set the `--short-circuit` flag along with `--all-valid`:
+```
+% ./bin/persephone.sh validate \
+  --profile sample_profiles/calibration.jsonld \
+  --statement sample_statements/calibration_1.json \
+  --short-circuit --all-valid
+```
+then only the first validation failure encountered will appear.
```

Comment:
This appears to actually be all validation failures for the first _template_ that fails validation.

Reply:
Yes that's exactly it, the first failing _template_ versus the first particular failure. I guess I can word the description better

Reply:
d30cb2aaf064edcfcde206c9fd2aa262792acbde